lib/MusicBrainz/Server/Data/Entity.pm

```diff
@@ -60,11 +48,18 @@ sub get_by_id
     return $result[0];
 }
 
-sub get_by_id_locked
-{
+sub get_by_id_locked {
     my ($self, $id) = @_;
-    my @result = values %{$self->_get_by_keys_append_sql($self->_id_column, 'FOR UPDATE', $id)};
```

Comment:
This was the only thing `_get_by_keys_append_sql` was used for. But the query it's modifying is too simple to be worth factoring this out, I think.